README.md

```diff
@@ -4,7 +4,23 @@ For users on MacOS, the Porter project maintains a <a href="https://brew.sh/">Ho
 
 <p align="center">Learn all about Porter at <a href="https://porter.sh">porter.sh</a></p>
 
-🚨 This isn't ready to use yet. We will update this page with the proper commands to run once our tap is complete!
+## Initial setup
+To tap Porter formula repository, execute following command.
+```
+brew tap getporter/porter
```

Comment:
Should this be `getporter/homebrew` since that is where the recipe is located?

Reply:
`brew tap getporter/porter` is cloning `github.com/getporter/hombrew-porter`.
Currently, `github.com/getporter/hombrew-porter` is redirected to `github.com/getporter/hombrew`.
So, if you want to execute `brew tap getporter/homebrew` the name of the repository should be `homebrew-homebrew`.

Reply:
Oh! Thank you I didn't understand that there is an assumed homebrew- prefix on the repository name.

I believe the redirect is happening because the repository name originally was getporter/homebrew-porter? Seems like it would be less confusing if the repository was named exactly what it is expecting without relying on the redirect.

Reply:
Indeed, I suggest to rename it back to `homebrew-porter`